lib/components/user/verify-email-screen.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { Button } from 'react-bootstrap'
+
+const _handleClick = () => window.location.reload()
+
+/**
+ * This component contains the prompt for the user to verify their email address.
+ * It also contains a button that lets the user finish account setup.
+ *
+ * (One way to make sure the parent page fetches the latest email verification status
+ * is to simply reload the page.)
+ */
+const verifyEmailScreen = (
```

Comment:
Did you ever look into the possibility of Auth0 redirecting the user to the OTP app after they click the email verification link? Also, this needs styling work (probably makes sense to just give all of these full screen views a classname that applies styling or otherwise use Bootstrap containers or something).

Reply:
Redirecting from Auth0's link should be handled separately.